Changes.md

```diff
@@ -15,10 +17,17 @@ Breaking Changes
 - FilteredChildIterator/FilteredRecursiveChildIterator : Removed all namespace-level typedefs, which were deprecated in Gaffer 0.59.0.0. Use the class-level typedefs instead, for example `Plug::Iterator` in place of `PlugIterator`.
 - Gadget :
   - Moved `render()` and `renderRequestSignal()` to ViewportGadget.
-  - Made `select()` private.
+  - Removed `select()` method.
+  - Replaced `hasLayer()` virtual function with `layerMask()`.
+  - Added `renderBound()` virtual function.
 - ViewportGadget : Deprecated old `gadgetsAt()` signature. Please use the new form instead.
 - ArnoldOptions : Removed support for the `ai:ignore_motion_blur` option. Turn off the `sampleMotion` option using a StandardOptions node instead.
 
+Improvements
+------------
+
+- Better UI performance when navigating large node graphs
```

Comment:
- GraphEditor : Improved drawing performance for large node graphs, \<insert some relevant numbers\>.